deployment/ingress/eks/ingress-controller.yaml

```diff
@@ -0,0 +1,652 @@
+apiVersion: v1
```

Comment:
Why do we need to maintain our own copy of the nginx helm chart templates? Could we use a stock ones via the deploy script using something like this?

`$ helm install my-ingress-controller nginx/nginx-ingress`

Reply:
@Voxelot - this is for specifically for AWS nginx ingress-controller and the specific annotations for AWS ingress controller 

the generic helm chart installs nginx-ingress that is not specific to aws and its load balancers

This article explains it better:

https://aws.amazon.com/blogs/opensource/network-load-balancer-nginx-ingress-controller-eks/